android/app/src/main/res/values-es-rPR/strings.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
+  <!-- Channel label for notifications shown in App > Notifications > Path Check BT -->
+  <string name="notification_channel_description">Notificaciones de PathCheck BT</string>
```

Comment:
@juancruzgs mentioned these removed so we'll run some cleanup soon on lokalise, no work on your part @alpita-masurkar